src/api/dataStoreApi/helpers/datastoreDomainToDomain.ts

```diff
@@ -0,0 +1,23 @@
+import { ethers } from "ethers";
+import { Domain } from "../../../types";
+import { DataStoreDomain } from "../types";
+
+export function datastoreDomainToDomain(d: DataStoreDomain): Domain {
+    const domain: Domain = {
+        id: d.domainId.toLowerCase(),
+        name: d.name,
+        parentId: d.parent.toLowerCase(),
+        owner: d.owner.toLowerCase(),
+        minter: d.minter.toLowerCase(),
+        metadataUri: d.metadataUri,
+        isLocked: d.locked,
+        lockedBy: d.lockedBy
+          ? d.lockedBy.toLowerCase()
+          : ethers.constants.AddressZero,
+        contract: d.registrar.toLowerCase(),
+        isRoot: d.isRoot,
+        buyNow: d.buyNow
```

Comment:
If the `Domain` type can have an undefined `buyNow` field, then maybe we make `buyNow` undefined if `d.buyNow.isActive` is false? It doesn't make too much sense to return a buy now price if it's not active. I don't think anyone would care to know the inactive buy now price, and if anything, it might be confusing.

Reply:
Good point, I see how that would just add confusion.